From f1811e543d75cf616c5b14fcca1d5b676a6023cd Mon Sep 17 00:00:00 2001
From: Sinthu Raja <sinthu.raja@ti.com>
Date: Thu, 16 Feb 2023 12:17:08 +0530
Subject: [PATCH 2/2] configs: j721s2: update config to select the right dtb
 for AM68 SK

Add condition to load the right dtb for AM68 SK based on the board
revision read from the eeprom.

Signed-off-by: Sinthu Raja <sinthu.raja@ti.com>
---
 include/configs/j721s2_evm.h | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/include/configs/j721s2_evm.h b/include/configs/j721s2_evm.h
index 2d26eb544d..00da2c0670 100644
--- a/include/configs/j721s2_evm.h
+++ b/include/configs/j721s2_evm.h
@@ -69,7 +69,9 @@
 		"setenv name_fdt ${default_device_tree};"		\
 		"if test $board_name = j721s2; then "			\
 			"setenv name_fdt k3-j721s2-common-proc-board.dtb; fi;" \
-		"if test $board_name = am68-sk; then "			\
+		"if test $board_name = am68-sk && test ${board_ver} = E1; then "			\
+			"setenv name_fdt k3-am68-sk-reve1-base-board.dtb; fi;"\
+		"if test $board_name = am68-sk && test ${board_ver} = E2; then "			\
 			"setenv name_fdt k3-am68-sk-base-board.dtb; fi;"\
 		"setenv fdtfile ${name_fdt}\0"				\
 	"name_kern=Image\0"						\
-- 
2.36.1

